index.rst

```diff
@@ -6,24 +6,474 @@
+how the user can make changes to the tutorial test options, or take the next step forward on their own.
+
+
+
+Tables and plots
+----------------
+
+Table data format
+^^^^^^^^^^^^^^^^^
+
+Results from a TAP service search are best displayed as an astropy table using ``.to_table()``,
+or as a pandas dataframe using ``.to_table().to_pandas()``.
+However, do not use the ``.to_table().show_in_notebook()`` method.
+This can cause issues in the RSP JupyterLab environment that make the notebook hang indefinitely.
+
+Plot color palette
+^^^^^^^^^^^^^^^^^^
+
+To be colorblind-friendly, plots should use the matplotlib color tables viridis or
+`cividis <https://matplotlib.org/stable/users/prev_whats_new/whats_new_2.2.html#cividis-colormap>`_ (or a greyscale),
+or the new `tableau-colorblind10 <https://viscid-hub.github.io/Viscid-docs/docs/dev/styles/tableau-colorblind10.html>`_. 
+
+:: 
+
+  import matplotlib.pyplot as plt
+  plt.style.use('tableau-colorblind10')
+
+
+For the LSST filter set ugrizy, adopt the same colors as DES, which were chosen to be colorblind-friendly: 
+
+::
+
+  plot_filter_colors = {'u': '#56b4e9', 'g': '#008060', 'r': '#ff4000', 'i': '#850000', 'z': '#6600cc', 'y': '#000000'}
+
+
+Image orientation
+^^^^^^^^^^^^^^^^^
+
+If using a WCS, display east left, north up.
+If only using pixels, (0,0) should be lower left, which is the default for awfDisplay.
+
+When using other plotting packages, transformations might be needed in order to match the afwDisplay default.
+See the LSST Science Pipelines documentation about `Image Indexing <https://pipelines.lsst.io/modules/lsst.afw.image/indexing-conventions.html>`_.
+
+Since use of "extent" is necessry for displaying a WCS overlay for deepCoaads, use it all the time:
+
+::
+
+  deepCoadd = butler.get('deepCoadd', dataId=dataId)
+  deepCoadd_bbox = butler.get('deepCoadd_calexp.bbox', dataId=dataId)
+  deepCoadd_wcs = butler.get('deepCoadd_calexp.wcs', dataId=dataId)
+  deepCoadd_WCSfMd = WCS(deepCoadd_wcs.getFitsMetadata())
+  deepCoadd_extent = (deepCoadd_bbox.beginX, deepCoadd_bbox.endX, deepCoadd_bbox.beginY, deepCoadd_bbox.endY)
+  plt.subplot(projection=deepCoadd_WCSfMd)
+  plt.imshow(deepCoadd.image.array, cmap='gray', vmin=0, vmax=2, extent=deepCoadd_extent, origin='lower')
+
+
+Clearing memory
+---------------
+
+These are optional methods for keeping memory use manageable in notebooks which are, e.g., demonstrating
+data visualization techniques with big datasets.
+
+To reduce the memory footprint of a notebook, remove figures once they're no longer needed.
+See the ``remove_figure`` function defined in the DP0 notebook `03_Image_Display_and_Manipulation.ipynb <https://github.com/rubin-dp0/tutorial-notebooks/blob/main/03a_Image_Display_and_Manipulation.ipynb>`__.
+
+*A method for clearing memory of, for example, large arrays that are not going to be used further on in the notebook is in development.*
```

Comment:
This note should be in a box or something similar.

Reply:
done